### PR TITLE
[KV-3429] Added Single IP Logic

### PR DESCRIPTION
**Pull request by Xxavicho**

### CODE REVIEW

**Comment by devkushkipagos** [comment]
@juan-ee: please review this pull request.

> Powered by [pull-review](https://github.com/imsky/pull-review)

File: kushki-android/src/test/java/com/kushkipagos/android/KushkiUnitTest.kt [hunk 1]
```diff
@@ -52,6 +54,18 @@ class KushkiUnitTest {
         assertThat(transaction.message, equalTo(errorMessage))
     }
 
+    @Test
+    @Throws(KushkiException::class)
+    fun shouldReturnOKMessageWhenCalledWithSingleIP() {
```
Comment (pmoreanoj) [inline kushki-android/src/test/java/com/kushkipagos/android/KushkiUnitTest.kt]: poner TODO cuando ya se tenga la url regional

**Pull request merged**